### ISSUE TITLE
Re: ver:   2.1.4   And  ver: 1.1.0

### ISSUE THREAD

**Issue by BoroBoros** [emeraldpay/emerald-wallet#1058]
Greetings,

I am a nuube so please excuse if this is in wrong format or place.

I am trying to setup a full ETC node on one of my mining rigs.  I see that Emerald Wallet ver:  1.1.0  did have the "Full Node" Option.  However, the newest ver: 2.1.4  has no (visible) option for Full Node.
Does Emerald no longer offer "Full Node"?  Please help me understand if Emerald can be a full node +wallet or just wallet?   In looking at ver 1.1.0 I see Emerald Wallet using Geth to do Full Node.  Is Geth going to be reintegrated into the new versions?

Thank you so much for your time and attention to this.

Kind Regards
BoroBoros

**Comment by splix** [comment]
Hi BoroBoros,

We changed the way how Emerald uses a Full Node since v1.3.0. Because now Emerald supports different blockchains, which can have different schedules for node upgrade, we stopped to ship a full node software with Emerald Wallet. Instead of that, it tries to use an existing Full Node if the user is running any. 

I.e., if you have software that runs Full Node on the same machine and it's running when Emerald is launched, the Emerald automatically connect it. Emerald Wallet can connect to Geth, Parity, running ETH, or ETC.
